### ISSUE TITLE
The applicant couldn't recognize the game in Ryujinx.(Mac OS)

### ISSUE THREAD

**Issue by MichaelFok** [Ecks1337/RyuSAK#39]
**Describe the bug**

**To Reproduce**
_When open the Ryusak , there were no games loading in the interface._

**Screenshots**

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/119941173/205910550-4783c70e-4594-4359-9d50-860c95f925a0.png">
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/119941173/205910659-8140bf88-cd88-4472-b603-f9cf785edb5f.png">

**Desktop (please complete the following information):**
- OS: Mac13.0 (22A380)
- Version 1.60

**Additional context**

- I have run the games in Ryujinx, such as FIFA23,  FIFA22 ,Mario strike. However, those games couldn't 
- be recognized in ryusak, Even I reinstalled ryusak.

**Comment by Rasesz** [comment]
Yep, me too. I had tried many times, always no games loading in the interface.
OS: Mac13.0.1 (22A400)
Version 1.60

**Comment by Xpl0itR** [comment]
Open Ryujinx, in the "File" menu click the option to open ryujinx folder. Send me the path to the folder that opens. Also in the folder there should be a sub folder called "games", in that folder, confirm that there are more folders with numbers as their names

**Comment by MichaelFok** [comment]
> Open Ryujinx, in the "File" menu click the option to open ryujinx folder. Send me the path to the folder that opens. Also in the folder there should be a sub folder called "games", in that folder, confirm that there are more folders with numbers as their names

hello, thank you for your replying 
I confirmed that " game " folders had named after numbers(maybe is the game id), 
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/119941173/205981750-2a557687-ea35-4df5-8ac7-
f4098c250aa8.png">


And my path to the folder is 
<img width="651" alt="image" src="https://user-images.githubusercontent.com/119941173/205982093-6eb8e985-de0d-4a41-8807-663b88f997f3.png">
<img width="594" alt="image" src="https://user-images.githubusercontent.com/119941173/205982226-5aebc22e-0780-4930-a1cb-00f764779a99.png">

I sincerely hope that my  Ryusak can be run normally ! LoL

**Comment by Rasesz** [comment]
> Open Ryujinx, in the "File" menu click the option to open ryujinx folder. Send me the path to the folder that opens. Also in the folder there should be a sub folder called "games", in that folder, confirm that there are more folders with numbers as their names

<img width="729" alt="2022-12-07 01 36 16" src="https://user-images.githubusercontent.com/46751858/205982711-61a2f0c7-ba88-4fec-9793-30428c2c98e0.png">

RyuSAK always prompt “... is reading data from". The path is right.

**Comment by Ecks1337** [comment]
Ok that's weird, it looks correct for both of you. Try clicking on the configuration dropdown and select the option to add additional configuration, then select the path to the Ryujinx folder you've shown me

**Comment by MichaelFok** [comment]
> Ok that's weird, it looks correct for both of you. Try clicking on the configuration dropdown and select the option to add additional configuration, then select the path to the Ryujinx folder you've shown me

I guess the app will scan the " game id " folder and print the game in interface.
So I copied a new folder "Ryujinx" and put the "games id" folders in it. And then I changed the path in RyuSak . But it still unworkable.
<img width="851" alt="截屏2022-12-07 08 18 28" src="https://user-images.githubusercontent.com/119941173/206057386-3fd2d266-1d83-4c97-a8db-3c8c61739228.png">

<img width="851" alt="截屏2022-12-07 08 18 40" src="https://user-images.githubusercontent.com/119941173/206057461-ae8191be-ac91-4a32-a561-63bdbe447071.png">




<img width="851" alt="截屏2022-12-07 08 18 28" src="https://user-images.githubusercontent.com/119941173/206057525-6159eae7-b361-43db-a697-2e59ebbed439.png">

**Comment by MichaelFok** [comment]
> Ok that's weird, it looks correct for both of you. Try clicking on the configuration dropdown and select the option to add additional configuration, then select the path to the Ryujinx folder you've shown me

I guess the app will scan the " game id " folder and print the game in interface.
So I copied a new folder "Ryujinx" and put the "games id" folders in it. And then I changed the path in RyuSak . But it still unworkable.

<img width="697" alt="截屏2022-12-07 08 30 19" src="https://user-images.githubusercontent.com/119941173/206057630-22da5aea-48a4-43bc-920d-effe8496cd39.png">


<img width="851" alt="截屏2022-12-07 08 18 28" src="https://user-images.githubusercontent.com/119941173/206057650-13c07876-2962-418b-88bf-ce79bf82ad04.png">




<img width="851" alt="截屏2022-12-07 08 18 40" src="https://user-images.githubusercontent.com/119941173/206057690-64dc680a-1ebf-48a7-9a71-6c95739c6400.png">

**Comment by MichaelFok** [comment]
To my surprise, my app can recognize the games right now. Just because I didn't close the ryusak , And it suddenly work well.
